### ISSUE TITLE
Add advices for training looking for tensorboard

### ISSUE THREAD

**Issue by Mercuron** [Unity-Technologies/ml-agents#5742]
I got some problems withmy own locomotion tasks. 
my own model is walking, but it can walk only directly and got troubles with rotation, but it can walk . i got big std of reward (40-50% of reward).
There is my tensorboard:
![bzp3lgu](https://user-images.githubusercontent.com/45468521/167266245-ad734693-039e-4d57-93a6-528ecce6545e.png)
![smsZWw1](https://user-images.githubusercontent.com/45468521/167266251-da02d611-d33e-429f-b0d9-c598ca9e25a0.png)
There is my config:
```
behaviors:
  SimpleWalker:
    trainer_type: ppo
    hyperparameters:
      batch_size: 2048
      buffer_size: 20480
      learning_rate: 0.0003
      beta: 0.005
      epsilon: 0.2
      lambd: 0.95
      num_epoch: 3
      learning_rate_schedule: linear
    network_settings:
      normalize: true
      hidden_units: 512
      num_layers: 3
      vis_encode_type: simple
    reward_signals:
      extrinsic:
        gamma: 0.995
        strength: 1.0
    keep_checkpoints: 30
    checkpoint_interval: 1000000
    max_steps: 50000000
    time_horizon: 1000
    summary_freq: 30000

```
1) Its would be nice for adding at documentaion some cases for best training. 
For example if your reward looks like this ... You should try to do this ...
2) Is my cumulative reward have problem called "Catastrophic forgetting" ? What are steps to remove it?
3) Is there any resources for tuning and read tensorboard? For example i got strange Entropy ,is it normal ? And what graphs should look like? 
What u can say looking at my tensorboard - is training stuck, is training got best what it can, or there is troubles with reward design?
Thanks !

**Comment by AKemendo** [comment]
Thanks for your questions @mercurion. Your cumulative reward graph isn't showing the hallmarks of catastrophic forgetting however it doesn't seem to be converging and your reward variance seems to be increasing. 

You will most likely get a faster answer with more detail if you ask these in the ML-Agents forum (https://forum.unity.com/forums/ml-agents.453/).

We will log your request for additional example documentation on training.

Thank you.

**Comment by Mercuron** [comment]
Judging by the number of responses here and on the forum, the answer will be received faster here (thank you).

As I understand it, there are no ready-made answers here how to act in this or that case? Do I need to act by trial and error only (increasing batch-size,buffer memory etc)?
p.s. what the last recommended version unity + MLagents ? my is 2020.3.25f1 and 2.1.0_exp-1?
